### PR TITLE
feat: Improve OCR regex

### PR DESCRIPTION
**Pull request by KobeW50**
This should make the bot respond to the screenshot shown below. I also made the response title consistent with other responses of the same issue.

image

I have no idea what I'm doing.

### CODE REVIEW

File: apps/bot-discord/src/config.json [hunk 1]
```diff
@@ -334,9 +334,9 @@
       }
     },
     {
-      "regex": "You're offline|Please check your",
+      "regex": "You're offline|Please check your|Couldn't sign in",
       "reply": {
-        "title": "Why does YouTube say, I am offline ❓",
+        "title": "Why does YT ReVanced say I am offline ❓",
```
Comment (oSumAtrIX) [inline apps/bot-discord/src/config.json:339]: This abbreviationand technically the name should not be used

```suggestion
        "title": "Why YouTube say I am offline after patching ❓",
```

**Comment by oSumAtrIX** [comment]
When is the other string displayed?

File: apps/bot-discord/src/config.json (hunk 1 above)
Comment (KobeW50) [reply apps/bot-discord/src/config.json:339]: "after patching" implies that the issue occurred immediately after patching.

How about `Why does YouTube ReVanced say I am offline?`?

**Comment by KobeW50** [comment]
> When is the other string displayed?

A few times in the telegram config.json.

https://github.com/ReVanced/revanced-helper/pull/28/files#diff-14996e806e27feff2ca1eb91ddeb1ef77a967069f937e9ea1db466db2832fbacR104-R105

https://github.com/ReVanced/revanced-helper/pull/28/files#diff-14996e806e27feff2ca1eb91ddeb1ef77a967069f937e9ea1db466db2832fbacR67-R68

https://github.com/ReVanced/revanced-helper/pull/28/files#diff-14996e806e27feff2ca1eb91ddeb1ef77a967069f937e9ea1db466db2832fbacR35

File: apps/bot-discord/src/config.json (hunk 1 above)
Comment (oSumAtrIX) [reply apps/bot-discord/src/config.json:339]: That's what it currently is.

Comment (KobeW50) [reply apps/bot-discord/src/config.json:339]: How about `Why does patched YouTube say I am offline ❓️`

**Comment by KobeW50** [comment]
@oSumAtrIX So the `config.example.json`s are now in place of the `config.json`s? I'm confused bec I'm not sure what is there as an example and what is actually being used
https://github.com/ReVanced/revanced-helper/commit/3b8e7ff5682333efd87d5a35fe82badc944b3c68

**Comment by oSumAtrIX** [comment]
No the `config.json` is read by the bots, whereas the example only acts as a reference.

**Comment by KobeW50** [comment]
but there's no config.json files anymore 
image

**Comment by oSumAtrIX** [comment]
Config files are environment-specific. They should not be present in source control.

**Comment by KobeW50** [comment]
Sorry, I hardly know what that means (I'm rather confused)

Does this mean it is impossible for me to PR changes to the bot responses and OCR? If it is still possible, how should/can I go about suggesting changes?

**Comment by oSumAtrIX** [comment]
You can not PR environment-specific variables or configs. These are handled outside of the VCS.

**Pull request closed without merging**